server/database/entities/users.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.orm import Session
+from uuid import UUID, uuid4
+import dataclasses
+from server.database.schemas.users import UserCreate
+
+
+@dataclasses.dataclass
+class User:
+    id: UUID
+    email: str
+    hashed_password: str
+    settings: str = "{}"
+
+    @classmethod
+    def create(cls, user: UserCreate):
+        hashed_password = user.password + "debug_not_really_hashed"
```

Comment:
I hope it wouldn't survive until deploying to production
It wouldn't be out of place to mark it with a comment like`# FIXME`

Reply:
Fixed